pkg/store/store.go

```diff
@@ -42,13 +42,16 @@ import (
 
 	"github.com/kong/kubernetes-ingress-controller/pkg/annotations"
 	kongv1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
+	kongv1alpha1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1alpha1"
 	"github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1beta1"
 	kongv1beta1 "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1beta1"
 )
 
 const (
-	knativeIngressClassKey = "networking.knative.dev/ingress.class"
-	caCertKey              = "konghq.com/ca-cert"
+	knativeIngressClassKey     = "networking.knative.dev/ingress.class"
+	caCertKey                  = "konghq.com/ca-cert"
+	ingressClassParamsAPIGroup = "configuration.konghq.com"
```

Comment:
You can probably use `kongv1alpha1.GroupVersion` instead of defining this const, no?